opencog/nlp/lg-dict/LGDictExpContainer.cc

```diff
@@ -199,47 +199,6 @@ void LGDictExpContainer::basic_normal_order()
 }
 
 /**
- * Output the scm representation of the LG dictionary entry.
- *
- * @return the scm string
- */
-std::string LGDictExpContainer::to_scm_string()
-{
-    if (m_type == CONNECTOR_type)
-    {
-        if (m_string == "OPTIONAL")
-            return "(LgConnector (LgConnectorNode \"0\"))\n";
-
-        std::stringstream ss;
-        ss << "(LgConnector (LgConnectorNode ";
-        ss << "\"" << m_string << "\") ";
-        ss << "(LgConnDirNode \"" << m_direction << "\") ";
```

Comment:
ah, well, as previously discussed, you could have just said `atomspace->newNode(LG_CON_DIOR_NDOE, m_direction);` here.